### PR TITLE
add java implementation to usaco uddered but not herd bronze

### PR DESCRIPTION
**Pull request by TruaShamu**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

NOTE: My bufferedreader line is over 80 characters :|

### CODE REVIEW

**Comment by TruaShamu** [comment]
proof it works lol
image

File: solutions/usaco-1083.mdx [hunk 1]
```diff
@@ -33,4 +33,32 @@ print(num_times)
 
 </PySection>
 
+<JavaSection>
+```java
+import java.io.*;
+
+public class udderedBronze {
```
Comment (nathangong) [inline solutions/usaco-1083.mdx:40]:
```suggestion
public class UdderedButNotHerd {
```

**Comment by nathangong** [comment]
Is this bronze or gold?

**Comment by TruaShamu** [comment]
> Is this bronze or gold?

bronze :)

**Comment by nathangong** [comment]
Ok u probably want to change the title of this PR then lol

**Comment by TruaShamu** [comment]
oops, sorry, it's fixed now

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-1083.mdx [hunk 2]
```diff
@@ -33,4 +33,29 @@ print(num_times)
...
+```java
+import java.io.*;
+
+public class UdderedButNotHerd {
+	public static void main(String[] args) throws IOException {
+		BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
+		String cowphabet = input.readLine();
+		char[] word = input.readLine().toCharArray();
+
+		int numTimes = 1;  // Every word must be hummed at least once.
```
Comment (SansPapyrus683) [inline solutions/usaco-1083.mdx:46]:
```suggestion
		int numTimes = 1;  // There must be at least one humming sequence.
```
idk feel like this is worded better

Comment (dongliu0426) [reply solutions/usaco-1083.mdx:46]: imo original is fine

Comment (SansPapyrus683) [reply solutions/usaco-1083.mdx:46]: i mean what does "word" mean?

Comment (TruaShamu) [reply solutions/usaco-1083.mdx:46]: uhhh
the string of lowercase letters that farmer john heard bessie say.

**Pull request merged**